chall06/vsempere.c

```diff
@@ -0,0 +1,218 @@
+#include <string.h>
+#include <stdlib.h>
+
+#define VSEMPERE_P 1
```

Comment:
Maybe use some more generic names? Like without your login, but this is a bit subjective.